backend/src/database/migrations/20220915123536-create-email-messages-transactional.js

```diff
@@ -11,14 +11,13 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.BIGINT,
       },
-      user_email: {
```

Comment:
haha I put in `user_email` instead of `user_id` so that we can look at the table rows to see who's sending it immediately (without having to look up the `users` table), but it's probably no big deal either way!